pkgs/applications/audio/opusfile/default.nix

```diff
@@ -1,23 +1,27 @@
 { lib, stdenv, fetchurl, pkg-config, openssl, libogg, libopus }:
 
 stdenv.mkDerivation rec {
-  name = "opusfile-0.12";
+  pname = "opusfile";
+  version = "0.12";
   src = fetchurl {
-    url = "http://downloads.xiph.org/releases/opus/${name}.tar.gz";
+    url = "http://downloads.xiph.org/releases/opus/${pname}-${version}.tar.gz";
     sha256 = "02smwc5ah8nb3a67mnkjzqmrzk43j356hgj2a97s9midq40qd38i";
   };
 
   nativeBuildInputs = [ pkg-config ];
   buildInputs = [ openssl libogg ];
   propagatedBuildInputs = [ libopus ];
-  patches = [ ./include-multistream.patch ];
+  patches = [ ./include-multistream.patch ]
+    # fixes problem with openssl 1.1 dependency
+    # see https://github.com/xiph/opusfile/issues/13
+    ++ lib.optionals stdenv.hostPlatform.isWindows [ ./disable-cert-store.patch ];
```

Comment:
Can we apply this unconditional?

Reply:
That github issue specifies where I found it was for windows compilation.

I'm not really sure if what it turns off is important (it might also be irrelevant for Nix use, I don't know)

Reply:
opusfile *builds* fine for linux with this patch in, but it seems like it turns off some feature that might be needed by some downstream packages.

Reply:
Yeah, we probably can't apply it for all platforms. Bummer.